lib/booker/v4/models/treatment_verified_bookable_online.rb

```diff
@@ -8,9 +8,6 @@ class TreatmentVerifiedBookableOnline < Model
                       'AvailableInAdvanceDateUnitType',
                       'Category',
                       'ColorCode',
-                      'CustomerRecordType',
```

Comment:
are these things just remnants of some unused stuff?

Reply:
Yes. We were going to put all of the fields in the metadata attribute in offer service responses, but members of the Booker team told us that a few of them are deprecated and/or not relevant.